news/_posts/releases/2020-04-20-release-note-1765.md

```diff
@@ -0,0 +1,102 @@
+
+As this patch fixes several security issues, we highly recommend to upgrade your shop as soon as possible. Of course, as always, don’t forget to backup before.
+
+Reminder:  the 1-Click Upgrade module’s latest version is [v4.10.1](https://github.com/PrestaShop/autoupgrade/releases/tag/v4.10.1), don’t forget to upgrade it.
+
+
+## Main fixes
+
+Below are listed the 7 regressions that were found and fixed in this version, impacting both front-office and back-office.
+
+Front-office regressions:
+
+- When editing an address both in the customer account and checkout, a new address was created instead of replacing it - [#18100](https://github.com/PrestaShop/PrestaShop/issues/18100) and [#18072](https://github.com/PrestaShop/PrestaShop/issues/18072)
+- Canonical redirects for products with combinations no longer worked, which could cause duplicate content [#18279](https://github.com/PrestaShop/PrestaShop/issues/18299)
+
+Back-office regressions:
+
+- When adding a cart rule to an order from the back-office, the value discount was not correct [#18630](https://github.com/PrestaShop/PrestaShop/issues/18630)
+- Searching a category with the quick search no longer redirected to the category edition page - [#17908](https://github.com/PrestaShop/PrestaShop/issues/17908)
+- The help card was no longer displayed on view order and new employee pages - [#18279](https://github.com/PrestaShop/PrestaShop/issues/18279) and [#18615](https://github.com/PrestaShop/PrestaShop/issues/18615)
+- In the customer view page, the number of “last emails” was incorrect - [#18602](https://github.com/PrestaShop/PrestaShop/issues/18602)
+- It was not possible to access the translation interface for the Serbian language - [#18062](https://github.com/PrestaShop/PrestaShop/issues/18062)
+
+
+## Security fixes
+
+Some security fixes have been included in this patch version to ensure an improved core reliability. Thanks a lot to Rabhi for finding a lot of these issues !
+
+Improper access controls:
+
+- on product page with combinations, attachments and specific prices ([security advisory](https://github.com/PrestaShop/PrestaShop/security/advisories/GHSA-cvjj-grfv-f56w))
+- on product attributes page ([security advisory](https://github.com/PrestaShop/PrestaShop/security/advisories/GHSA-4wxg-33h3-3w5r))
+- on customers search ([security advisory](https://github.com/PrestaShop/PrestaShop/security/advisories/GHSA-r6rp-6gv6-r9hq))
+- on several other pages ([security advisory](https://github.com/PrestaShop/PrestaShop/security/advisories/GHSA-74vp-ww64-w2gm))
+
+Reflected XSS:
+
+- related in import page ([security advisory](https://github.com/PrestaShop/PrestaShop/security/advisories/GHSA-98j8-hvjv-x47j))
```

Comment:
is related necessary ? or should we just write "on import page" ?

Reply:
@PierreRambaud

Reply:
Not necessary, you can do both